sources/install/package_c2.sh

```diff
@@ -16,18 +16,13 @@ function install_metasploit() {
     # CODE-CHECK-WHITELIST=add-aliases
```

Comment:
Please add `add-history` to the whitelist
Should be `# CODE-CHECK-WHITELIST=add-aliases,add-history`